airbyte-integrations/connectors/source-okta/integration_tests/configured_catalog_incremental.json

```diff
@@ -0,0 +1,14 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "users",
```

Comment:
Why do you have `users` here and also in `configured_catalog.json` file? Please merge these files into one and use this one file in tests.

Reply:
Done